### PR TITLE
Scenario for QA setup 2c: HA with SBD, ceph, and kvm compute

### PR DESCRIPTION
**Pull request by jsuchome**
This is the scenario file for QA setup 2c: HA with SBD, ceph, and kvm compute nodes.

1. Install the nodes with these mkcloud options:

  hacloud=1
  clusterconfig="data=2"
  nodenumber=7
  want_node_roles=controller=2,compute=2,storage=3
  want_node_os=suse-12.1=4,suse-12.0=3
  want_node_aliases=controller=2,compute=2,ceph=3

2. Setup SBD using https://gist.github.com/jsuchome/4a658676db6ea29e924f

3. Run crowbar batch with updated scenario file:
    crowbar_batch --timeout 1200 build qa-scenario-2c-sbd-kvm.yaml

### CODE REVIEW

**Comment by vmoravec** [comment]
Is this scenario file is a generic one? And should it work with any cloud after the nodes have been installed and aliases have been set up?

**Comment by jsuchome** [comment]
It's meant for https://trello.com/c/xB01S8EV/532-3-s31p5-s30p14a-qa-infra-mkcloud-ha-sbd-scenario-with-kvm-ceph-config-2c

So when HA is prepared (SBD mostly) and correct number of nodes, than it should work

File: scripts/scenarios/cloud6/qa-scenario-2c-sbd-kvm.yaml [hunk 1]
```diff
@@ -0,0 +1,213 @@
...
+    elements:
+      rabbitmq-server:
+      - cluster:services
+- barclamp: keystone
+  attributes:
+    ssl:
+      generate_certs: true
+      insecure: true
+    api:
+      protocol: https
```
Comment (vuntz) [inline scripts/scenarios/cloud6/qa-scenario-2c-sbd-kvm.yaml]: This is missing the UUID bits instead of PKI.

**Comment by vuntz** [comment]
+1

**Comment by vuntz** [comment]
Ah, maybe squash the new commit...

**Comment by vuntz** [comment]
+1

**Comment by aplanas** [comment]
+1

**Pull request merged**